core/src/main/java/org/elasticsearch/search/aggregations/pipeline/PipelineAggregatorFactory.java

```diff
@@ -158,6 +158,10 @@ public final XContentBuilder toXContent(XContentBuilder builder, Params params)
         return builder.endObject();
     }
 
+    protected boolean overrideBucketsPath() {
```

Comment:
can you add some javadocs? it's not entirely obvious to me what it does at first sight